src/util/StringStream.js

```diff
@@ -73,9 +74,14 @@ class StringStream {
   current(){return this.string.slice(this.start, this.pos)}
   hideFirstChars(n, inner) {
```

Comment:
> Another solution is adding this to the soy mode

If things can be fixed without touching the StringSteam API, I'd much prefer that.